js/components/grid/calendar.js.jsx

```diff
@@ -89,7 +100,11 @@ class TimetableHeader extends React.Component {
         <thead>
           <tr>
             <th className="timetable-dummy-cell"></th>
-            <th className="term-name">Fall</th>
+            <th className="term-name">
+              <MapModal ref={ r => this.modal = r}/>
```

Comment:
This is okay, but we should use the new form of refs now: https://reactjs.org/docs/refs-and-the-dom.html